scripts/commit-filter-check.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Get the current branch and apply it to a variable
+currentbranch=`git branch | grep \* | cut -d ' ' -f2`
+
+# Gets the commits for the current branch and outputs to file
+git log $currentbranch --pretty=format:"%H" --not master > shafile.txt
+
+# loops through the file an gets the message
+for i in `cat ./shafile.txt`;
+do 
+  # gets the git commit message based on the sha
+  gitmessage=`git log --format=%B -n 1 "$i"`
+
+  # fix(commit-filter-check): add commit messages (AEROGEAR-038928990423)
+   messagecheck=`echo $gitmessage | grep -w "feat\|fix\|docs" | grep "(AEROGEAR-" | grep "): "`
```

Comment:
This does not seem to be working for me. I am testing it with `feat: sdasdsadsaas (AEROGEAR-asdsada)`

Reply:
I would expect that to fail doesn't match the format
` feat(some txt): some txt (AEROGEAR-sometxt)`
Have a test repo on my own for different commits https://github.com/austincunningham/-git-commit-messages-filtering.git, checkout the branchname, didn't want to clutter production with test commit messages.

Reply:
The scope section is optional, i would have expected the above to pass.

Reply:
As its optional then no point checking will remove the check